### PR TITLE
chore: use releaseinfo.json to get the latest stable release

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

**Comment by HarikrishnanBalagopal** [comment]
Partial fix for https://github.com/konveyor/move2kube/issues/155

File: scripts/install.sh [hunk 1]
```diff
@@ -78,12 +79,18 @@ verifySupported() {
 
 # getLatestVersion gets the latest release version.
 getLatestVersion() {
-    # Get tag from release URL
-    local latest_release_url='https://github.com/konveyor/move2kube/releases'
+    # Get tag from releaseinfo.json
+    local json_data=''
+    local release_info_url='https://raw.githubusercontent.com/konveyor/move2kube/gh-pages/_data/releaseinfo.json'
```
Comment (ashokponkumar) [inline scripts/install.sh:84]: Is there no way to access the url using konveyor.io url as github pages?

**Review by ashokponkumar — COMMENTED** [review]
Also, we wanted to install konveyor_translate binary too. Do we want to install it as part of this PR or a new PR?

File: scripts/install.sh (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply scripts/install.sh:84]: why is that important?

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**